warehouse/accounts/models.py

```diff
@@ -163,6 +166,17 @@ def recent_events(self):
             .all()
         )
 
+    @property
+    def can_reset_password(self):
+        return not any(
+            [
+                self.is_superuser,
+                self.is_moderator,
+                self.is_psf_staff,
+                self.prohibit_password_reset,
```

Comment:
I not that you didn't change the name of the column. I guess this is voluntary?

Reply:
This is configured via admin only. So from the administrators perspective, flipping the bit prohibits password reset for a user. In this case it would be "sentinel" users like `admin` and `pypi` which we _do not_ want to grant any special privileges.

Reply:
Ok, it makes sense